internal/deb/extract.go

```diff
@@ -25,6 +25,7 @@ type ExtractOptions struct {
 	TargetDir string
 	Extract   map[string][]ExtractInfo
 	Globbed   map[string][]string
+	FSCreator *fsutil.Creator
```

Comment:
This might be just `Creator`, given the context, and although this is minor we can use a locally defined interface to allow for variation on the types, even more for testing purposes if required.

Reply:
Agree on the interface but can we do it when the need arises? Right now the tests depend on the default creator and the tree dump function that inspects the filesystem.